File: tools/web/goshop/js/service/brandService.js
//服务层
var url='http://localhost:8999/shopping-sellergoods-ms/brand-Ms';
app.service('brandService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get(url+'/findAll');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get(url+'/findPage?pageNum='+page+'&pageSize='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get(url+'/findBrandDetail?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post(url+'/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post(url+'/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get(url+'/delete?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post(url+'/search?page='+page+"&rows="+rows, searchEntity);
	}    
	//下拉列表数据
	this.selectOptionList=function(){
		return $http.get(url+'/selectOptionList');
	}
	
});

File: shopping-sellergoods-ms/src/main/java/com/online/shopping/service/BrandService.java
package com.online.shopping.service;

import java.util.List;
import java.util.Map;

import com.online.shopping.entity.PageResult;
import com.online.shopping.pojo.TbBrand;

/**
 * 品牌接口
 * @author Administrator
 *
 */
public interface BrandService {

	public List<TbBrand> findAll();
	
	/**
	 * 品牌分页
	 * @param pageNum 当前页面
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	/**
	 * 增加 
	 * @param brand
	 */
	public void add(TbBrand brand);
	
	/**
	 * 根据ID查询实体
	 * @param id
	 * @return
	 */
	public TbBrand findOne(Long id);
	
	/**
	 * 修改
	 * @param brand
	 */
	public void update(TbBrand brand);
	
	
	/**
	 * 删除
	 * @param ids
	 */
	public void delete(Long[] ids);
//	
//	
//	/**
//	 * 品牌分页
//	 * @param pageNum 当前页面
//	 * @param pageSize 每页记录数
//	 * @return
//	 */
//	public PageResult findPage(TbBrand brand, int pageNum,int pageSize);
	
	/**
	 * 返回下拉列表数据
	 * @return
	 */
	public List<Map> selectOptionList();
}
